src/docs/files/devguide06.md

```diff
@@ -28,9 +28,9 @@ Additionally, Auctioneers can further customize the auction by using the followi
 
 **&quot;--min-buy-amount-per-order&quot;** Describes the minimal buyAmount per order placed in the auction. Auctioneers can define if there is minimum order size in their auction. Default is 0.01
 
-**&quot;--is-atomic-closure-allowed&quot;** This parameter enables users to close the auction atomically and submit a final bid by calling the `settleAuctionAtomically` function in the smart contract once the `auction-end-date` has been reached. The auctioneer determines whether this parameter is on or off by specifying True or False.  
+**&quot;--is-atomic-closure-allowed&quot;** This parameter enables users to close the auction atomically and submit a final bid by calling the `settleAuctionAtomically` function in the smart contract once the `auction-end-date` has been reached. The auctioneer determines whether this parameter is on or off by specifying True or False.
 
-**&quot;--allow-list-manager&quot;** If whitelisting of addresses is required, the auctioneer can provide his/her own allowlistManager contract, or use the AllowListOffChainManaged.json file provided in [this repo](https://github.com/gnosis/ido-contracts/tree/main/deployments). The command needs to be followed by the contract address of the allowlistManager being used. 
+**&quot;--allow-list-manager&quot;** If whitelisting of addresses is required, the auctioneer can provide his/her own allowlistManager contract, or use the AllowListOffChainManaged.json file provided in [this repo](https://github.com/gnosis/ido-contracts/tree/main/deployments). The command needs to be followed by the contract address of the allowlistManager being used.
```

Comment:
instead of using "If whitelisting of address is required" use "If an Allow List of addresses is required, where only approved users will be able to participate, the auctioneer..."